python/ccf/clients.py

```diff
@@ -152,6 +152,7 @@ def json(self):
         return json.loads(self._data)
 
 
+# TODO: Merge seqno/view into unique tx_id
```

Comment:
Not sure this is necessary here, since we would like to remove this wrapper altogether.